hoomd/operation.py

```diff
@@ -374,7 +353,7 @@ class Compute(Operation):
     pass
 
 
-class Tuner(Operation):
+class Tuner(TriggeredOperation):
```

Comment:
If we are going to have `Tuner`s be `TriggeredOperation`s, then each tuner instance should call a super class constructor to set the trigger given to them in their constructors.

Reply:
Like this: c170f3f?